src/components/Row/Row.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import * as Styled from './Row.styled';
+import { FlexDivProps } from '../FlexDiv/FlexDiv.styled';
+
+const Row: React.FC<Styled.RowProps & FlexDivProps> = (props) => {
+  const { children } = props;
+  return (
+    <Styled.Row wrap="wrap" {...props}>
```

Comment:
On this code `{...props}` we are also sending `'children'` value... I wonder if is this what we want? 🤔